backend/tests/libs/util_db.py

```diff
@@ -23,6 +31,21 @@ def get_resource(resource_id):
         return None
 
 
+def modify_issue(issue_id, issue_property, value):
+    try:
+        issue = db.IssueProperty.filter(
+            IssueProperty.issue_id == issue_id, IssueProperty.name == issue_property
+        ).first()
+        if not issue:
+            return False
+        else:
+            issue.value = value
+            db.session.commit()
+            return True
```

Comment:
would we not want some logging if the modification fails instead of just rolling back?

Reply:
Added. Thanks!